handbook/engineering/README.md

```diff
@@ -250,6 +252,18 @@ Our goal is to keep these dependencies up-to-date with each release of Fleet. If
 
 Next, create a new GitHub issue using the [Release QA template](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=&projects=&template=smoke-tests.md&title=). Add the release version to the title, and assign the quality assurance members of the [MDM](https://fleetdm.com/handbook/company/development-groups#mdm-group) and [CX](https://fleetdm.com/handbook/company/development-groups#customer-experience-group) product groups.
 
+### Merging during the freeze period
+
+We merge bug fixes and documentation changes during the freeze period, but we do not merge other code changes. This minimizes code churn and helps ensure a stable release. To merge a bug fix, you must first unfreeze the PR in [Merge Freeze](https://app.mergefreeze.com/installations/3704/branches), and click the "Unfreeze 1 pull request" text link. 
+
+#### Exceptions
```

Comment:
This is a generic heading called "Exceptions", which doesn't make for the clearest URL:

```
fleetdm.com/handbook/engineering#exceptions
```

(like, what kind of exceptions?)

I think this heading can be removed.  In this case, the context of the content is clear based on its "Merging during the freeze period" heading.

Reply:
```suggestion

```